client/components/LearnerViewDirective.js

```diff
@@ -318,6 +324,21 @@ tie.directive('learnerView', [function() {
         .tie-previous-instructions {
           opacity: 0.5;
         }
+        .tie-reinforcement li {
+          list-style: none;
+          margin: 0;
+          margin-top: 1px;
+          position: relative;
+        }
+        .bullet-img {
+          width: 15px;
```

Comment:
Nit: recommend alphabetizing these attrs (i.e. bottom, then height, then position, then width). As the number of attributes increases, this makes it easier to locate a particular one.